store/rootmulti/store.go

```diff
@@ -972,6 +972,26 @@ func (rs *Store) buildCommitInfo(version int64) *types.CommitInfo {
 	}
 }
 
+// RollbackToVersion delete the versions after `target` and update the latest version.
+func (rs *Store) RollbackToVersion(target int64) int64 {
+	latest := getLatestVersion(rs.db)
+	i := latest
```

Comment:
```suggestion
	i := getLatestVersion(rs.db)
```

Reply:
the `latest` is used later now, to check if needed to update the latest height.